dali/operators/audio/preemphasis_filter_op.cu

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/audio/preemphasis_filter_op.h"
+#include <vector>
+
+namespace dali {
+
+namespace detail {
+
+template <typename OutputType, typename InputType>
+struct SampleDescriptor {
+  const InputType *in;
+  OutputType *out;
+  float coeff;
+  int64_t size;
+};
```

Comment:
Not really - this sample descriptor is copied as a whole and all threads in the warp get the same sample_desc - so distributing the fields in SoV instead of VoS would cause _more_ cache misses - this struct is so small that it generates two cache misses at most - and only if it crosses cache line boundary.